documentation/general/general_setup.md

```diff
@@ -184,6 +184,19 @@ If not, you can use one of the many
 [installation methods](https://docs.docker.com/engine/install/) officially
 supported by Docker.
 
+Note that the building images for the frontend needs an advanced build feature,
+namely additional build contexts, that is provided by
+[Buildx](https://docs.docker.com/build/architecture/#buildx). Buildx is included
+in Docker Desktop and all
+[installation methods](https://docs.docker.com/engine/install/) for Docker
+Engine include steps to also install the Buildx plugin `docker-buildx-plugin`.
```

Comment:
Is `docker-buildx-plugin` the package name? If so, this differs by distro and I think the link to the documentation is better suited for that, or a generic instruction to search for your distro's installation instructions.

Reply:
I had made sure that the name was the same for all distros before referring to it in the docs.

Fedora:
<img width="762" alt="Screenshot 2023-05-08 at 8 01 55 AM" src="https://user-images.githubusercontent.com/16580576/236731105-e5b1d00c-5617-4fdc-886b-864eaa050317.png">

CentOS/RHEL:
<img width="763" alt="Screenshot 2023-05-08 at 8 01 03 AM" src="https://user-images.githubusercontent.com/16580576/236731026-a4372bb5-502c-4d37-b67c-ac026a4f0482.png">

Debian/Ubuntu:
<img width="758" alt="Screenshot 2023-05-08 at 8 01 24 AM" src="https://user-images.githubusercontent.com/16580576/236731061-4e3b7b78-d19c-47b8-baf9-3d6d46215fe1.png">

SLES:
<img width="764" alt="Screenshot 2023-05-08 at 8 02 49 AM" src="https://user-images.githubusercontent.com/16580576/236731194-28cfde71-fe92-4757-b087-17b175436d99.png">

Reply:
For Arch it's `docker-buildx`. But of course, anyone not using one of those distros probably already knows to look elsewhere. In any case, repeating the implementation detail here is strange, and it could easily go out of date and cause confusion.